$NetBSD: patch-ae,v 1.5 2001/06/29 17:19:08 wiz Exp $

If these lines stay, `make install; make deinstall; make install' fails.

--- Makefile.1.orig	Fri Jun 29 04:20:17 2001
+++ Makefile.1
@@ -101,14 +101,12 @@
      then $(MANCOMPRESS) -c new/$$a.5 >$(MAN5DIR)/$$a.$(MAN5SUFFIX); \
      else :; fi; \
   done
-	echo Housekeeping file >install.man
 
 install.bin: $(BINS) $(BASENAME)
 	@-$(MKDIRS) $(BINDIR) 2>$(DEVNULL); exit 0
 	@chmod 0755 $(BINS)
 	$(INSTALL) $(BINS) $(BINDIR)
 	@-dirname / >$(DEVNULL) || $(INSTALL) examples/dirname $(BINDIR)
-	echo Housekeeping file >install.bin
 
 install:
 	@$(MAKE) install.man install.bin
